tests/codebuild/run_integtest.sh

```diff
@@ -1,33 +1,58 @@
 #!/bin/bash
 
-source tests/codebuild/common.sh
-
 # TODOs
 # 1. Add validation for each steps and abort the test if steps fails
 # Build environment `Docker image` has all prerequisite setup and credentials are being passed using AWS system manager
+# crd_namespace is currently hardcoded here for cleanup. Should be an env variable set in the pipeline and .env file
+
+source tests/codebuild/common.sh
+crd_namespace="test-namespace"
 
 # Verbose trace of commands, helpful since test iteration takes a long time.
 set -x 
 
-# A function to delete cluster, if cluster was not launched this will fail, so test will fail ultimately too
-function cleanup {
-    # We want to run every command in this function, even if some fail.
-    set +e
-
+# A function to print the operator logs for a given namespace
+# Parameter:
+#    $1: Namespace of the operator
+function get_manager_logs {
+    local crd_namespace="${1-sagemaker-k8s-operator-system}"
     if [ "${PRINT_DEBUG}" != "false" ]; then
-        echo "Controller manager logs:"
-        kubectl -n sagemaker-k8s-operator-system logs "$(kubectl get pods -n sagemaker-k8s-operator-system | grep sagemaker-k8s-operator-controller-manager | awk '{print $1}')" manager
+        echo "Controller manager logs in the ${crd_namespace} namespace"
+        kubectl -n "${crd_namespace}" logs "$(kubectl get pods -n "${crd_namespace}" | grep sagemaker-k8s-operator-controller-manager | awk '{print $1}')" manager
     fi
+}
+
+# A function that cleans up Jobs, CRDs and Operator in the default namespace
+function cleanup_default_namespace {
+    set +e
+    get_manager_logs
+    delete_all_resources "default"
+    kustomize build $path_to_installer/config/default | kubectl delete -f -
```

Comment:
++ $path_to_installer

nit: Wrap all variables in double quotes: `"${path_to_installer}/config/default"`

Reply:
done